packages/data/src/world/mq/bottom_of_the_well_mq.yml

```diff
@@ -18,39 +18,21 @@
 "Bottom of the Well Main":
   dungeon: BotW
   exits:
-    "Bottom of the Well Wallmaster Main": "soul_wallmaster && (can_play_zelda || (has_explosives_or_hammer && has_hover_boots))"
-    "Bottom of the Well Wallmaster Basement": "soul_wallmaster"
-    "Bottom of the Well Wallmaster Pit": "soul_wallmaster && (can_hit_triggers_distance || can_boomerang || has_explosives || can_hookshot)"
-    "Bottom of the Well": "true"
+    "Bottom of the Well": "is_child"
+    "Bottom of the Well Main Room Center": "event(BOTW_MQ_CENTER_GATES_LOWERED)"
+    "Bottom of the Well Main Room Center Switch Corner": "has_explosives_or_hammer"
```

Comment:
What about climb or hook anywhere? i imagine at least hook would work.

Reply:
climb does, hook does not. fixed.

Reply:
Hook does. include it.

Reply:
Added.